client/my-sites/stats/stats-navigation/index.jsx

```diff
@@ -34,7 +34,7 @@ const StatsNavigation = ( props ) => {
 			statsControl = (
 				<SegmentedControl
 					// eslint-disable-next-line wpcalypso/jsx-classname-namespace
-					className="stats-navigation__control"
+					className="stats-navigation__control for-woocommerce"
```

Comment:
These styles won't show up unless you go to store stats first due to the async load (if I'm not mistaken)